src/Symfony/Component/PropertyInfo/Tests/Fixtures/PseudoTypesDummy.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Symfony\Component\PropertyInfo\Tests\Fixtures;
+
+class PseudoTypesDummy
+{
+    /** @var class-string */
```

Comment:
`class-string` is usually used as a generic type, e.g. `class-string<SomeBaseClass>`. Does this make a difference for your implementation? Should we add a test case for that?

Reply:
Thanks for suggestion! After adding tests I discovered what I believe was a bug - `class-string<SomeBaseClass>` was extracted by PhpstanExtractor as a collection. I fixed it